stable/fairwinds-insights/values.yaml

```diff
@@ -552,8 +552,9 @@ minio:
   nameOverride: fw-minio
   persistence:
     # -- Create a persistent volume for Minio
-    enabled: true
-
+    enabled: false
+  replicas: 1
+  mode: standalone
```

Comment:
I don't think we should change the default value for `persistence` here. Pagerduty may be relying on it.

Reply:
done